python/requirements-wheel-build.txt

```diff
@@ -1,3 +1,8 @@
+# Remove pre and extra index url once there's NumPy and Cython wheels for 3.13
+# on PyPI
```

Comment:
Can we open a GH issue for this?

Reply:
I listed a bunch of follow-ups in the parent issue (apart from those cleanups, not everything is addressed by this PR), which I was planning to keep open and have labeled as 18.0 to ensure we do this before the next release.

(can open a sub-issue to have a distinct GH number for this PR)

Reply:
> I listed a bunch of follow-ups in the parent issue (apart from those cleanups, not everything is addressed by this PR), which I was planning to keep open and have labeled as 18.0 to ensure we do this before the next release.

Sounds good, thank you!